sentry-spring/src/main/java/io/sentry/spring/tracing/SentryTracingFilter.java

```diff
@@ -120,9 +120,6 @@ private ITransaction startTransaction(
             .log(SentryLevel.DEBUG, "Failed to parse Sentry trace header: %s", e.getMessage());
       }
     }
-    final ITransaction transaction =
-        hub.startTransaction(name, "http.server", customSamplingContext);
-    hub.configureScope(scope -> scope.setTransaction(transaction));
-    return transaction;
```

Comment:
@maciejwalkowiak can you confirm that this should be removed? `startTransaction` already does it automatically.

Reply:
Yep, thats correct, missed it.